src/modules/twap/pure/ConfirmDetailsItem/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { ReactNode } from 'react'
+
+import { CornerDownRight } from 'react-feather'
+
+import { RowFixed } from 'legacy/components/Row'
+import { MouseoverTooltipContent } from 'legacy/components/Tooltip'
+
+import { TextWrapper } from 'modules/swap/pure/Row/styled'
+import { StyledInfoIcon } from 'modules/swap/pure/styled'
+
+import { Wrapper, Row, Content } from './styled'
+
+type Props = {
```

Comment:
why not `ConfirmDetailsItemProps` and exporting? (just curious)

Reply:
This part is coming from original Nenad's PR.
Will rename